libsolidity/grammar.txt

```diff
@@ -35,14 +35,20 @@ Continue = 'continue' ';'
 Break = 'break' ';'
 Return = 'return' Expression? ';'
 Throw = 'throw' Expression? ';'
-VariableDefinition = VariableDeclaration ( = Expression )? ';'
+VariableDefinition = VariableDeclaration ( '=' Expression )? ';'
 
 Expression = Assignment | UnaryOperation | BinaryOperation | FunctionCall | NewExpression | IndexAccess |
-			 MemberAccess | PrimaryExpression
+             MemberAccess | PrimaryExpression
 // The expression syntax is actually much more complicated
 Assignment = Expression (AssignmentOp Expression)
+BasicBinaryOperation = '|' | '^' | '&' | '<<' | '>>' | '>>>' | '+' | '-' | '*' | '/' | '%'
```

Comment:
These were actually not included on purpose, because the parsing is much more complicated than expressed here.